docs/user-guide/get-started.md

```diff
@@ -1,13 +1,11 @@
 # Getting started
 
-Stylelint is designed for CSS.
+You can lint:
 
-However, it can be used with [PostCSS syntaxes](https://github.com/postcss/postcss#syntaxes) that:
+- CSS files by using our standard config
+- everything else by using extensions written by the community
```

Comment:
[suggest] For example, to clarify "everything else", what if adding examples like this?

```suggestion
- everything else by using extensions written by the community (e.g., HTML, JSX, etc.)
```

Reply:
I rethought it. The following section explains "everything else" in detail, so such examples are needless. Sorry.

Reply:
No worries. I forgot to mention that I'm assuming most users will arrive on this page having just read the README/homepage that gives examples of this too: https://stylelint.io/